HousingSearchApi/V1/Boundary/Requests/GetProcessListRequest.cs

```diff
@@ -12,7 +12,13 @@ public class GetProcessListRequest : HousingSearchRequest
         public Guid? TargetId { get; set; }
 
         [FromQuery(Name = "isOpen")]
-        public bool IsOpen { get; set; }
+        public bool? IsOpen { get; set; }
+
+        [FromQuery(Name = "processName")]
+        public string ProcessName { get; set; }
+
+        [FromQuery(Name = "patchId")]
```

Comment:
Hi Humairaa - can you please confirm if searchText will be used to query by patchId or if this separate property is needed?

Reply:
SearchText will be used for patchid, will remove this